frontend/src/libs/components/input/input.tsx

```diff
@@ -70,9 +70,7 @@ const Input = <T extends FieldValues>({
         disabled={isDisabled}
         inputMode={inputMode}
       />
-      <span className={styles['error-message']}>
-        {hasError && (error as string)}
-      </span>
+      <ErrorMessage error={error as string} />
```

Comment:
I think you have no need to cast it to string

Reply:
if i notadd this, i will have error
Type 'string | ("message" extends keyof FieldError & keyof DeepRequired<T>[string] ? [FieldError["message" & keyof DeepRequired<T>[string]], FieldErrorsImpl<...>["message" & keyof DeepRequired<...>[string]]] extends [...] ? Merge<...> : FieldError["message" & keyof DeepRequired<...>[string]] | FieldErrorsImpl<...>["messag...' is not assignable to type 'string | undefined'.